Week3/homework/step2-1.js

```diff
@@ -1,9 +1,7 @@
 'use strict';
 
 function foo(func) {
-  // What to do here?
-  // Replace this comment and the next line with your code
-  console.log(func);
+  return func();
```

Comment:
You may, but do not have to return the return value of `func` as the return value of `foo`. In this case that return value will be `undefined` as `bar` does not return a value:

```js
function foo(func) {
  func();
}
```

Reply:
Got it.